test/bigtest/tests/loan-actions-history-test.js

```diff
@@ -46,6 +54,42 @@ describe('loans actions history', () => {
     expect(LoanActionsHistory.closeButton.isPresent).to.be.true;
   });
 
+  describe('display Overdue Fine Policy and Lost Item Fee Policy', () => {
+    it('fields should be presented', () => {
+      expect(LoanActionsHistory.overduePolicy.isPresent).to.be.true;
+      expect(LoanActionsHistory.lostItemPolicy.isPresent).to.be.true;
+    });
+
+    it('should display the field name', () => {
+      expect(LoanActionsHistory.overduePolicy.text).to.equal('One Hour1');
+      expect(LoanActionsHistory.lostItemPolicy.text).to.equal('One Hour2');
+    });
+
+    describe('clicking the overdue policy link', () => {
+      beforeEach(async function () {
+        await LoanActionsHistory.overduePolicy.click();
+        await this.visit(`/settings/circulation/fine-policies/${openLoan.overdueFinePolicyId}`);
```

Comment:
I am not sure the visit call should be made. The idea is that after the click on the button the navigation occurs according to the logic you have written. So there is no need to make a visit programmatically in tests. The code should do it instead.

The same [here](https://github.com/folio-org/ui-users/pull/1103/files#diff-0c6e69ff7433118d17d13645a1ec126bR83).

Reply:
I agree, it sounds correct, url shoud not be changed manually